packages/react-ds/src/components/Button/Button.tsx

```diff
@@ -14,6 +14,7 @@ interface IButtonProps extends IButton {
   endIcon?: React.ReactNode | string
   /** React Element | string */
   startIcon?: React.ReactNode | string
+  onClick: () => void
```

Comment:
I think we could use this:
```suggestion
  onClick: (event: React.MouseEvent<HTMLButtonElement>): void
```